entries/h/heimdalsecurity.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Heimdal": {
+    "domain": "heimdalsecurity.com",
+    "url": "https://www.heimdalsecurity.com/",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.heimdalsecurity.com/hc/en-us/articles/212721589-Set-up-the-2-Factor-Authenticator",
```

```suggestion
    "documentation": "https://support.heimdalsecurity.com/hc/articles/212721589",
```
Just to make it a little tidier ;)